packages/cms/pages/resources/resource/resource.tsx

```diff
@@ -192,28 +193,17 @@ export const Table: React.FunctionComponent = () => {
   const [selectedItems, setSelectedItems] = useState<any[]>([])
   const [sortField, setSortField] = useState('firstName')
   const [sortDirection, setSortDirection] = useState<'asc' | 'desc'>('asc')
+  const { resource, applyFilter } = useResourceStore()
 
   const columns: EuiBasicTableColumn<any>[] = [
-    {
-      field: 'firstName',
-      name: 'First name',
-      sortable: true,
-      truncateText: true
-    },
-    {
-      field: 'lastName',
-      name: 'Last name',
-      sortable: true,
-      truncateText: true
-    },
-    {
-      name: 'Date of birth',
-      field: 'dateOfBirth'
-    },
-    {
-      name: 'Nationality',
-      field: 'nationality'
-    },
+    ...(resource?.fields
+      .filter((resource, idx) => resource.showOnIndex && idx != 0)
```

Comment:
Quick question, why `idx != 0`? Also, in future, it's much recommended to check using `!==` if you need to. Its a much more exact search and better for code predictability.

Reply:
I noticed the ID field was not shown in the example, so I filtered it. I can put it back if it is required.